jekyll/_cci2/migrating-from-travis.md

```diff
@@ -33,6 +33,31 @@ configuration will live in a `.travis.yml` file in the root of your repository.
+| language:         | [docker.image](https://circleci.com/docs/2.0/configuration-reference/#docker)                | Use the Docker executor to specify an appropriate Docker image for the target language            |
+| dist:             | [docker, machine, macos](https://circleci.com/docs/2.0/executor-types/)      | If your build must run in a fully virtualized environment, or on a macOS builder                  |
+| cache components: | [restore_cache:,](https://circleci.com/docs/2.0/configuration-reference/#restore_cache) [save_cache:](https://circleci.com/docs/2.0/configuration-reference/#restore_cache) | Use the restore and save cache features to control caching in the builds                          |
+| before_cache      | [restore_cache:,](https://circleci.com/docs/2.0/configuration-reference/#restore_cache) [save_cache:](https://circleci.com/docs/2.0/configuration-reference/#restore_cache) | Use the restore and save cache features to control caching in the builds                          |
+| before_install:   | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                        | Use the run: step to specify pre install commands                                                 |
+| install:          | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                        | Use the run: step to specify install commands                                                     |
+| before_script     | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                        | Use the run: step to specify pre-execution commands                                               |
+| script:           | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                        | Use the run: step to specify execution commands                                                   |
+| after_script:     | [run:](https://circleci.com/docs/2.0/configuration-reference/#run)                        | Use the run: step to specify post-execution commands                                              |
+| deploy:           | [deploy:](https://circleci.com/docs/2.0/configuration-reference/#deploy)                     | Use the deploy: step to deploy build artifacts                                                    |
+| env:              | [environment:](https://circleci.com/docs/2.0/configuration-reference/#environment)                | Use the environment: element to specify environment variables                                     |
+| matrix:           | [workflows:](https://circleci.com/docs/2.0/configuration-reference/#workflows)                  | Workflows are used on CircleCI to orchestrate multiple jobs                                       |
+| stage:            | [requires:](https://circleci.com/docs/2.0/configuration-reference/#requires)                   | Use the requires: element to explicitly require any job dependencies and control parallel builds. |
+
```

Comment:
actually need to add this line at end of the table for formatting:

{: class="table table-striped"}

Reply:
Done 👍